src/check_aggregator_impl.cc

```diff
@@ -14,19 +14,15 @@ limitations under the License.
 ==============================================================================*/
 
 #include "src/check_aggregator_impl.h"
-#include "src/signature.h"
-
 #include "google/protobuf/stubs/logging.h"
+#include "src/signature.h"
 
-using std::string;
 using ::google::api::MetricDescriptor;
-using ::google::api::servicecontrol::v1::Operation;
 using ::google::api::servicecontrol::v1::CheckRequest;
 using ::google::api::servicecontrol::v1::CheckResponse;
-using ::google::protobuf::util::OkStatus;
-using ::google::protobuf::util::Status;
-using ::google::protobuf::util::StatusCode;
+using ::google::api::servicecontrol::v1::Operation;
 using ::google::service_control_client::SimpleCycleTimer;
+using std::string;
```

Comment:
nit: Is this some we want to remove given use of both string & std::string in this file?

Reply:
I think so. It is not ideal usage but I don't plan to change in this PR.